File: TestNG/Activities/Activity1.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity1 {
    //craete webdriver
    private WebDriver driver ;
    @BeforeClass
    public void setUp(){
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriverManager.firefoxdriver().setup();
        driver = new FirefoxDriver();
        driver.get("https://www.training-support.net");
    }
    @Test
    public void checkOriginalTitle(){
        String title = driver.getTitle();
        Assert.assertEquals("Training Support",title);

    }
    @Test
    public void clickAboutUsButton(){
        WebElement button = driver.findElement((By.id("about-link")));
       Assert.assertNotNull(button);
       button.click();

    }
    @AfterClass
    public void teardown(){
        driver.quit();
    }

}

File: Python/Activities/Activity1.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service = service) as driver:
  
  # Open the browser to the URL
  driver.get("https://training-support.net")
  print(f"title of the page is" ,driver.title)
  driver.find_Element(By.ID,"about-link").click()  
  driver.quit()
